content/applications/finance/accounting/taxes/avatax.rst

```diff
@@ -1,80 +1,405 @@
+:menuselection:`Accounting --> Configuration --> Settings` section. The :guilabel:`AvaTax` fields in
+the :guilabel:`Taxes` section is where the *AvaTax* configurations are made and the credentials are
+entered in.
 
 .. image:: avatax/avatax-configuration-settings.png
    :align: center
-   :alt: Configure Avatax settings
+   :alt: Configure AvaTax settings
 
-.. _avatax/tax-mapping:
+Prerequisites
+~~~~~~~~~~~~~
 
-Tax mapping
-===========
+First, select the :guilabel:`Environment` that the company wishes to use AvaTax in. It can either be
+:guilabel:`Sandbox` or :guilabel:`Production`.
 
-The Avatax integration is available on Sale Orders and Invoices with the included Avatax fiscal
-position.
+.. seealso::
+   For help determining which *AvaTax* environment to use (either :guilabel:`Production` or
+   :guilabel:`Sandbox`), visit: `Sandbox vs Production environments
+   <https://knowledge.avalara.com/bundle/fzc1692293626742/page/sandbox-vs-production.html>`_.
 
-Before using the integration, specify an :guilabel:`Avatax Category` on the product categories.
+Credentials
+~~~~~~~~~~~
 
-.. image:: avatax/avatax-category.png
-   :align: center
-   :alt: Specify Avatax Category on products
+Now, the credentials can be entered in. The AvaTax :guilabel:`Account ID` will be entered in the
+:guilabel:`API ID` field and the :guilabel:`License Key` will be entered in the :guilabel:`API Key`
+field.
 
-Avatax Categories may be overridden or set on individual products as well.
+.. important::
+   The :guilabel:`Account ID` can be found by logging into the AvaTax portal (`sandbox
+   <https://sandbox.admin.avalara.com/>`_ or `production <https://admin.avalara.com/>`_). In the
+   upper-right corner, click on the initials of the user and :guilabel:`Account`. The
+   :guilabel:`Account ID` is listed first.
 
-.. image:: avatax/override-avatax-product-category.png
+   To access the :guilabel:`License Key` see this documentation:
+   :ref:`avatax/create_avalara_credentials`.
+
+For the :guilabel:`Company Code` field enter the Avalara company code for the company being
+configured. Avalara interprets this as `DEFAULT` if it is not set. The :guilabel:`Company Code` can
+be accessed in the Avalara management portal.
+
+First, log into the AvaTax portal (`sandbox <https://sandbox.admin.avalara.com/>`_ or `production
+<https://admin.avalara.com/>`_). Then, navigate to :menuselection:`Settings --> Manage Companies`.
+The :guilabel:`Company Code` value is located in the row of the :guilabel:`Company` in the
+:guilabel:`Company Code` column.
+
+.. image:: avatax/company-code.png
    :align: center
-   :alt: Override product categories as needed
+   :alt: AvaTax company code highlighted on the company details page.
+
+Transaction options
+~~~~~~~~~~~~~~~~~~~
+
+There are two transactional settings in the AvaTax settings in Odoo that can be configured:
+:guilabel:`Use UPC Commit` and :guilabel:`Commit Transactions`.
```

Comment:
```suggestion
:guilabel:`Use UPC` and :guilabel:`Commit Transactions`.
```
correct me if I'm wrong, but it appears that these two fields look like my suggestion -- although, I can see how the current formatting of them on the Settings page makes it bit difficult to discern. Feel free to disregard if I'm mistaken with my suggestion.